interface/ceed-elemrestriction.c

```diff
@@ -401,13 +401,15 @@ int CeedElemRestrictionApplyBlock(CeedElemRestriction rstr, CeedInt block,
   @brief Get the multiplicity of nodes in a CeedElemRestriction
 
   @param rstr             CeedElemRestriction
+  @param lmode            Ordering of the ncomp components
```

Comment:
Should we have a more complete description, like the one we have in `CeedElemRestrictionApply` for instance? 
```
Ordering of the ncomp components, i.e. it specifies
                   the ordering of the components of the l-vector used
                   by this CeedElemRestriction. CEED_NOTRANSPOSE indicates
                   the component is the outermost index and CEED_TRANSPOSE
                   indicates the component is the innermost index in
                   ordering of the l-vector
```

Reply:
I have no strong feelings either way, so I added the more verbose description

Reply:
Good. This is what I've had floating for way too long in my branch: fa9eac48825b2c0339cddc576d44c186940d4520